config/routes.rb

```diff
@@ -1,6 +1,5 @@
 Rails.application.routes.draw do
-  devise_for :users, :controllers => { registrations: 'registrations'}
   resources :posts
```

Comment:
- [ ] Restrict routes  in posts using resources :posts, only: [:new, :create, :index]
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_routes_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication))